tools/cli/go-whisk-cli/commands/qualified_name.go

```diff
@@ -28,7 +28,7 @@ type QualifiedName struct {
     namespace   string  // namespace. does not include leading '/'.  may be "" (i.e. default namespace)
     packageName string  // package.  may be "".  does not include leading/trailing '/'
     entity      string  // entity.  should not be ""
-    entityName  string  // pkg+entity
+    EntityName  string  // pkg+entity
```

Comment:
Hmm, not sure I understand this change.

Reply:
@dubeejw This was needed in code in another repo where we do an http call for getting/updating an Action. Based on existing code here, the call to the REST method is made with the `EntityName` of a QualifiedName struct.  Since this are private to this repo, I was not able to use it. Was not trying to duplicate code which creates the QualifiedName struct or how the REST calls are made.